.gitignore

```diff
@@ -11,9 +10,21 @@ camel/app.ini
 camel/test.txt
 config/app.ini
 *.txt
-*.ini
+/data.ini
+/dests.ini
+/app.ini
 ile/src/mzversion.h
 camel/src/main/java/com/github/theprez/manzan/Version.java
 appinstall.jar
-manzan-installer-*.jar
-.env
+manzan-installer-*.jar
+.env
+
+.logs

```

Comment:
Where are these entries from?

Reply:
.env file contains some entries used for testing. Description can be found in readme.

.logs and manzan-installer-*.jar are just files that were in my directory. Removed now.